meta-oe/recipes-kernel/ipmitool/ipmitool_1.8.19.bb

```diff
@@ -27,10 +27,12 @@ SRC_URI = "git://github.com/ipmitool/ipmitool;protocol=https;branch=master \
            file://0001-ipmi_fru.c-Provide-missing-function-declarations.patch \
            file://0001-configure-Remove-the-logic-to-download-IANA-PEN-data.patch \
            "
-IANA_ENTERPRISE_NUMBERS ?= ""
 
-# Add these via bbappend if this database is needed by the system
-#IANA_ENTERPRISE_NUMBERS ?= "http://www.iana.org/assignments/enterprise-numbers;name=iana-enterprise-numbers;downloadfilename=iana-enterprise-numbers"
+# Default IANA enterprise database version is 2022-10-24
+IANA_ENTERPRISE_NUMBERS ?= "file://iana-enterprise-numbers.tar.xz"
```

Comment:
Can we host this tarball somewhere and download it during build ? git does not do well with binary blobs particularly. I am sure we will bump revisions of this as well in future.

Reply:
thanks for your reply ;p
Indeed, it's a bit hard to manage with git, the information in tarball from the IANA site, it's a pure text file to record vender/enterprise database the size is almost 4.4MB,  it seems a bit large so I created a tarball.

Reply:
I think we can keep a default enterprise database to make sure ipmitool is working normal, if a user wants to use the latest version database, they can add a bbappend file to turn on "IANA_ENTERPRISE_NUMBERS = "http://www.iana.org/assignments/enterprise-numbers;name=iana-enterprise-numbers;downloadfilename=iana-enterprise-numbers"" in their own layer.

Reply:
> Can we host this tarball somewhere and download it during build ? git does not do well with binary blobs particularly. I am sure we will bump revisions of this as well in future.

thanks for your reply ;p
Indeed, it's a bit hard to manage with git, the information in tarball from the IANA site, it's a pure text file to record vender/enterprise database the size is almost 4.4MB, it seems a bit large so I created a tarball.

I think we can keep a default enterprise database to make sure ipmitool is working normal, if a user wants to use the latest version database, they can add a bbappend file to turn on "IANA_ENTERPRISE_NUMBERS = "http://www.iana.org/assignments/enterprise-numbers;name=iana-enterprise-numbers;downloadfilename=iana-enterprise-numbers"" in their own layer.

thanks,
Br,
Xiangyu